src/hooks/useLoadClientScript.ts

```diff
@@ -19,15 +20,46 @@ export function useLoadClientScript(props: NostoScriptProps) {
       }
       setClientScriptLoaded(true)
     }
-
-    // Create script element
-    function createScriptElement(urlPartial: string) {
+  
+    // Create and append script element
+    function injectScriptElement(urlPartial: string, language?: string, marketId?: string) {
```

Comment:
replace language and marketId with

`extraAttributes?: Record<string, string>`

that way it's more general